### PR TITLE
Typo fixing in the README_zh.md file

### PR DESCRIPTION
**Pull request by Lnek**
### What type of PR is this?
/kind documentation

### What this PR does / why we need it:
Typo fixing in the README_zh.md file
### Special notes for reviewers:
```
```

### Does this PR introduced a user-facing change?

```
None
```

### Additional documentation, usage docs, etc.:

```

```

### CODE REVIEW

File: README_zh.md [hunk 1]
```diff
@@ -60,7 +60,7 @@
 
 <details>
   <summary><b>🔗 Kubernetes 多集群管理</b></summary>
-  提供集中控制平台来管理多个 Kubernetes 集群，支持将应用程序发布到跨不同云供应商的多个k8集群上。
+  提供集中控制平台来管理多个 Kubernetes 集群，支持将应用程序发布到跨不同云供应商的多个k8s集群上。
```
Comment (FeynmanZhou) [inline README_zh.md:63]: 多个 K8s 集群上。

**Review by FeynmanZhou — CHANGES_REQUESTED** [review]
Thanks for your PR! Please see my comments : )

**Review by LinuxSuRen — COMMENTED** [review]
Thanks for your contribution. But please review the PR template again.

image

**Comment by LinuxSuRen** [comment]
/retitle Typo fixing in the README_zh.md file

**Comment by FeynmanZhou** [comment]
@Lnek Could you update it according to the comment above?

**Comment by Lnek** [comment]
@FeynmanZhou done

**Pull request merged**